### ISSUE TITLE
Failed to acquire CRAS

### ISSUE THREAD

**Issue by JeremiahHsu** [dnschneid/crouton#3123]
```
Please paste the output of the following command here: sudo edit-chroot -all
```
chronos@localhost / $ sudo edit-chroot -all
name: precise
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/precise...
crouton: version 1-20170228132702~master:21cb695b
release: precise
architecture: amd64
targets: kde
host: version 9000.91.0 (Official Build) stable-channel swanky 
kernel: Linux localhost 3.10.18 #1 SMP Wed Feb 22 23:32:43 PST 2017 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/precise...

#### Please describe your issue:
Cannot complete chroot setup due to failing fetching CRAS.

#### If known, describe the steps to reproduce the issue:
I was longing to install Ubuntu on my Toshiba CB35, and after a really long list of something, the system started to acquire CRAS.
"Connecting to chromium.googlesource.com (chromium.googlesource.com)|2404:6800:4008:c06::52|:443... failed: Network is unreachable."
and for unknown reason,the process was terminated and all the effort seemed gone.
I typed sudo startkde as a hail Mary. Unlike previous attempts, the system came back with"the chroot may not be fully set up, would you like to finish it?", it liked a ray of hope that broke the darkness.
With joy and excitement I typed yes. Yet the system still cannot acquire CRAS persistently. I copied the link and found it's accessible in Chrome, and the file has been successfully downloaded. 
Anyhow, I tried to boot the system again but this time it came back with "UID 1000 not found in precise",as my last ray of hope faded away.
I have established a proxy link in System Setting due to the censorship in China mainland. However it doesn't solve my issue.
Thank you for your reading and I hope God be with you.

**Comment by JeremiahHsu** [comment]
I found a modification that requires a mandatory SSL link in order to ensure proxy involved on Chinese website. 
I tried with that and it came back with "Proxy tunneling failed: Proxy Authentication Required Unable to establish SSL connection."
I have deployed a Shadowsocks on another computer and used as a proxy server link via LAN, it works fine until now. On other website no security issue discovered. That computer using AES-256 to establish link with some server (you guys call it VPS right?)
